Add Analyzer & Codefix - CA1847 -  Use String.Contains(Char) instead of String.Contains(String) with single characters (#4908)
dotnet/roslyn-analyzers: src/NetAnalyzers/Microsoft.CodeAnalysis.NetAnalyzers.md

|CodeFix|True|
---

> ## [CA1841](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1841): Use 'string.Contains(char)' instead of 'string.Contains(string)' with single characters

Reviewer: ```suggestion
## [CA1841](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1841): Use char literal for a single character lookup
```
